src/Sentry/buildTransitive/Sentry.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Helpful properties used elsewhere -->
+  <PropertyGroup Condition="'$(TargetFramework)' != ''">
+    <TargetFrameworkVersion>$([MSBuild]::GetTargetFrameworkVersion($(TargetFramework)))</TargetFrameworkVersion>
```

Comment:
_potentially dangerous ... I know we used the name `TargetFrameworkVersion` previously already ... this has slipped through an earlier review_

`TargetFrameworkVersion` is a Common MSBuild project property.
User or other MSBuild code may depend on it.
See https://learn.microsoft.com/visualstudio/msbuild/common-msbuild-project-properties
And we essentially remove the `v` prefix from this property.

Perhaps we should prefix this property with `Sentry`.
Perhaps we should prefix all properties, that we inject into consuming builds with `Sentry`.

This would make the `IsNet8OrGreater` slightly longer, but I think it's worth it to not override any potentially user-defined properties.